### PR TITLE
RavenDB-12511 Expose queries and how to kill them in the studio

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

**Comment by ml054** [comment]
cc: @arekpalinski

**Review by ppekrol — APPROVED** [review]

**Review by arekpalinski — APPROVED** [review]

File: src/Raven.Server/Documents/Handlers/Debugging/ServerWideQueriesDebugHandler.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+using System.Threading.Tasks;
+using Raven.Server.Documents.Queries;
+using Raven.Server.Routing;
+using Raven.Server.Web;
+
+namespace Raven.Server.Documents.Handlers.Debugging
+{
+    public class ServerWideQueriesDebugHandler : RequestHandler
+    {
+        [RavenAction("/debug/queries/running/live", "GET", AuthorizationStatus.ValidUser)]
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/Debugging/ServerWideQueriesDebugHandler.cs:14]: No, this is wrong.
This is server wide, and we have to make sure that this is:
`/admin/debug` and that the permission level is `Operator`

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Documents/Handlers/Debugging/ServerWideQueriesDebugHandler.cs (hunk 1 above)
Comment (ml054) [reply src/Raven.Server/Documents/Handlers/Debugging/ServerWideQueriesDebugHandler.cs:14]: We have single endpoint for every user, please have a look at https://github.com/ravendb/ravendb/pull/8436/files/2bb1368e4f0037cc356ebf5798659055073ce222#diff-e1eadb52a3455ec6c3d3be2636d09b4eR16  we filter to allowed databases. So you get only allowed db in this websocket. 

Is it ok then?

File: src/Raven.Server/Documents/Queries/LiveRunningQueriesCollector.cs [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+using Raven.Server.ServerWide;
+using Raven.Server.Utils.Stats;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Queries
+{
+    public class LiveRunningQueriesCollector : LivePerformanceCollector<LiveRunningQueriesCollector.ExecutingQueryCollection>
+    {
+        private readonly ServerStore _serverStore;
+        private readonly string[] _dbNames;
```
Comment (ayende) [inline src/Raven.Server/Documents/Queries/LiveRunningQueriesCollector.cs]: Turn this to a Set, instead.

File: src/Raven.Server/Documents/Handlers/Debugging/ServerWideQueriesDebugHandler.cs (hunk 1 above)
Comment (ayende) [reply src/Raven.Server/Documents/Handlers/Debugging/ServerWideQueriesDebugHandler.cs:14]: yes

File: src/Raven.Server/Documents/Queries/LiveRunningQueriesCollector.cs (hunk 2 above)
Comment (ml054) [reply src/Raven.Server/Documents/Queries/LiveRunningQueriesCollector.cs]: done

**Pull request merged**